conans/client/generators/cmake_common.py

```diff
@@ -608,56 +632,21 @@ def generate_targets_section(dependencies):
         endforeach()
     endif()
 endmacro()
-
-macro(conan_flags_setup)
```

Comment:
Is there any harm in leaving this macro? Even if not documented, some users might be using it, is it really necessary to remove it now?

Reply:
I think it is a very old macro no one should be using, but yes, I can keep it